lib/likadan_uploader.rb

```diff
@@ -36,4 +37,17 @@ def upload_diffs
     html.save
     html.url
   end
+
+  private
+
+  def find_or_build_bucket
+    bucket = service.buckets.find(BUCKET_NAME)
```

Comment:
Looks like you'll need to pass `service` as an argument to this method in order for this to work.